### PR TITLE
Update images for title cards on the community page

### PR DESCRIPTION
**Pull request by nick-mon1**
Replaces **CoP**  card images with color bar branding. More details https://trello.com/c/2wwdQTs6

**Current**

current-state

**After**

Screen Shot 2022-07-11 at 2 41 56 PM

### CODE REVIEW

File: themes/digital.gov/layouts/communities/list.html [hunk 1]
```diff
@@ -40,24 +40,16 @@ <h1>{{- .Title | markdownify -}} </h1>
             {{ range $i, $e := $dg_communities }}
 
             <li class="usa-card tablet:grid-col-4">
```
Comment (mejiaj) [inline themes/digital.gov/layouts/communities/list.html:42]: If we're going to be using these colors for communities, we should tie them to their colors so it's easier to maintain. You could add a class to each card to differentiate. Then you could look through them in sass.

For example a classes like `cop--fcn` or `cop--federal-communicators-network`. In your SASS you'd have:

```scss
$communities-of-practice: (
  "federal-communicators-network": "#fa9441",
  "multilingual": "#d73a33",
  …
);

@each $community, $color in $communities-of-practice {
  .usa-card.cop--#{community} {
    border-top: #{$color}
  }
}
```

This way you can easily update colors and add/remove communities.

File: themes/digital.gov/layouts/communities/list.html [hunk 2]
```diff
@@ -40,24 +40,16 @@ <h1>{{- .Title | markdownify -}} </h1>
...
-                      aria-hidden="true"
-                    />
-                  </div>
-                </div>
                 <div class="usa-card__body">
                   <p>{{- $e.Params.summary | markdownify -}}</p>
                 </div>
                 <div class="usa-card__footer">
-                  <a class="usa-button" href="{{- $e.Permalink -}}" title="{{- $e.Title -}}">
+									<a class="usa-button" href="{{- $e.Permalink -}}" title="{{- $e.Title -}}">
```
Comment (mejiaj) [inline themes/digital.gov/layouts/communities/list.html:52]: Can we add some hover/active/focus states to these buttons?

**Review by mejiaj — COMMENTED** [review]
Looking good, just some minor suggestions. 

I'm not _opposed_ to the decorative `div` element for the bar, but I think we can achieve the same and have clearer code if we used a pseudoelement or a border.

File: content/communities/federal-communicators-network.md [hunk 3]
```diff
@@ -34,8 +34,6 @@ weight: 1
 
 # Spotlight Digital.gov Communities of Practice (COP) at top of /communities
 dg_highlight: true
-dg_image: img/dg-card-1.png
```
[the remark this answers is not in the document]
Comment (ToniBonittoGSA) [reply content/communities/federal-communicators-network.md:37]: Yes, please

[the remark this answers is not in the document]
Comment (nick-mon1) [reply content/communities/federal-communicators-network.md:37]: Images are removed

File: themes/digital.gov/layouts/communities/list.html [hunk 4]
```diff
@@ -40,24 +40,16 @@ <h1>{{- .Title | markdownify -}} </h1>
             {{ range $i, $e := $dg_communities }}
 
             <li class="usa-card tablet:grid-col-4">
-              <div class="usa-card__container">
+							<div class="usa-card__container">
+								<div class="usa-card__cop-bar"></div>
```
[the remark this answers is not in the document]
Comment (nick-mon1) [reply themes/digital.gov/layouts/communities/list.html]: I've applied the `border-top` to `usa-card__container`

File: themes/digital.gov/layouts/communities/list.html (hunk 1 above)
Comment (nick-mon1) [reply themes/digital.gov/layouts/communities/list.html]: Refactored code to the suggested pattern

File: themes/digital.gov/layouts/communities/list.html (hunk 2 above)
Comment (nick-mon1) [reply themes/digital.gov/layouts/communities/list.html:51]: Reverted back to existing button styles

**Comment by ToniBonittoGSA** [comment]
Looks great https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.app.cloud.gov/preview/gsa/digitalgov.gov/nl-cop-update-cards/communities/

**Pull request merged**